packages/toolkit/src/hooks/useError/index.js

```diff
@@ -0,0 +1,8 @@
+import { useCallback } from 'react';
+
+export default function useError() {
+  return useCallback((error) => {
+    console.error('Error detected !!!!!!');
```

Comment:
I don't think we will put the sentry on the sdk, so perhaps we can remove this hook.. what do you think?

Reply:
We could put Sentry on the SDK with an Opt-in param and a dedicated dsn for customers.